webmails/roundcube/start.py

```diff
@@ -60,52 +61,56 @@
 context["PLUGINS"] = ",".join(f"'{p}'" for p in plugins)
 
 # add overrides
-context["INCLUDES"] = sorted(inc for inc in os.listdir("/overrides") if inc.endswith((".inc", ".inc.php"))) if os.path.isdir("/overrides") else []
```

Comment:
We should document it somewhere if we change this

Reply:
I started the PR before the new includes logic. The includes logic with both extensions should be used. I will change this back